app/components/create-vhost/template.hbs

```diff
@@ -20,59 +20,122 @@
-                        {{#bs-tooltip placement="bottom" title='Custom endpoints let you use your own certificate and domain name.' bs-container=false}}
-                            Use a custom endpoint address
-                        {{/bs-tooltip}}
+                      {{#radio-button value=TYPE_GENERIC groupValue=vhostType name="domain-type"}}
+                        Use a custom domain and a custom SSL certificate.
                       {{/radio-button}}
                     </div>
                 </div>
               </div>
-          {{/unless}}
 
             <div class="form-group">
-              <label class="block" for="vhost-type">Type</label>
+              <label class="block" for="vhost-type">
+                Endpoint Placement
+                {{more-info-icon
+                  title="Choosing an Endpoint Placement"
+                  bs-html=true
+                  body="
+                    <p>
+                    <strong>External endpoints</strong> are accessible via the
+                    a public internet. Use them for user-facing applications.
+                    </p>
+                    <p>
+                    <strong>Internal endpoints</strong> are only accessible by
+                    other apps in your private Aptible infrastructure. Use them
+                    for internal APIs.
+                    </p>"}}
+              </label>
+              {{#if placementNeeded}}
               <div class="radio">
                 <div>
                   {{#radio-button value=false groupValue=model.internal name="vhost-type"}}
-                    {{#bs-tooltip placement="bottom" title='External endpoints are accessible from the outside internet.' bs-container=false}}
-                        External
-                    {{/bs-tooltip}}
+                    External
                   {{/radio-button}}
                 </div>
                 <div>
-                  {{#radio-button value=true groupValue=model.internal name="vhost-type"}}
-                    {{#bs-tooltip placement="bottom" title='Internal endpoints are only accessible from other apps in your stack.' bs-container=false}}
-                        Internal
-                    {{/bs-tooltip}}
-                  {{/radio-button}}
+                    {{#radio-button value=true groupValue=model.internal name="vhost-type" }}
+                      Internal
+                    {{/radio-button}}
                 </div>
               </div>
+            {{else}}
+              <p>
+                This Endpoint type does not support internal endpoints. The
+                endpoint will be external.
+              </p>
+            {{/if}}
             </div>
 
-          {{#unless model.isDefault}}
+          {{#if acmeDomainNeeded}}
+            {{domain-input domain=model.acmeDomain valid=acmeDomainValid
+              tip="You'll neeed to be able to create a CNAME from this domain
```

Comment:
(@fancyremarker I pushed this for now; but let me know if you'd prefer something else)